extensions/purchase-label/functions/src/index.ts

```diff
@@ -37,8 +39,13 @@ export const purchaseLabel = functions.handler.firestore.document.onWrite(
       handleUpdateDocument(change.after, update);
     } catch (err) {
       // Update the document with error information on failure
-      if ((err as Error).message)
-        handleUpdateDocument(change.after, { error: (err as Error).message });
+      if ((err as Error).message) {
+        handleUpdateDocument(change.after, {
+          [config.shippingLabelKey]: {
+            errors: (err as Error).message,
```

Comment:
This is unrelated to the purpose of this PR, but I realized I wasn't putting the errors in the correct key.